apps/emqx_retainer/src/emqx_retainer.appup.src

```diff
@@ -1,15 +1,14 @@
-%% -*-: erlang -*-
+%% -*- mode: erlang -*-
 {VSN,
-  [
-    {<<"4\\.3\\.[0-1]+">>, [
-      {load_module, emqx_retainer, brutal_purge, soft_purge, []}
-    ]},
-    {<<".*">>, []}
-  ],
-  [
-    {<<"4\\.3\\.[0-1]+">>, [
-      {load_module, emqx_retainer, brutal_purge, soft_purge, []}
-    ]},
-    {<<".*">>, []}
-  ]
-}.
+  [{"4.3.2",
+    [{load_module,emqx_retainer_cli,brutal_purge,soft_purge,[]}]},
+   {<<"4\\.3\\.[0-1]+">>,
```

Comment:
Nit: maybe this could be more strict?  It's matching `10`, `11`, etc, but those are yet to exist.